src/CombFilter/CombFilterIf.cpp

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
As per my understanding of the assignment, we were given the header file for an interface class, and based on that, we had to create an underlying class, and implement the interface functions to call the underlying class methods. Here you have not implemented anything within the interface class cpp file.